app/src/androidTest/java/org/mozilla/fenix/ui/robots/TabDrawerRobot.kt

```diff
@@ -489,32 +492,27 @@ private fun syncedTabsButton() = onView(withContentDescription("Synced tabs"))
 private fun newTabButton() = mDevice.findObject(UiSelector().resourceId("$packageName:id/new_tab_button"))
 private fun threeDotMenu() = onView(withId(R.id.tab_tray_overflow))
 
-private fun assertExistingOpenTabs(title: String) {
-    try {
-        mDevice.findObject(
-            UiSelector()
-                .resourceId("$packageName:id/mozac_browser_tabstray_title")
-                .textContains(title)
-        )
-            .waitForExists(waitingTime)
-
-        assertTrue(
-            mDevice.findObject(
-                UiSelector()
-                    .resourceId("$packageName:id/mozac_browser_tabstray_title")
-                    .textContains(title)
-            ).waitForExists(waitingTime)
-        )
-    } catch (e: AssertionError) {
-        println("The tab wasn't found")
-        mDevice.findObject(UiSelector().resourceId("$packageName:id/tabsTray")).swipeUp(2)
-        assertTrue(
-            mDevice.findObject(
-                UiSelector()
-                    .resourceId("$packageName:id/mozac_browser_tabstray_title")
-                    .textContains(title)
-            ).waitForExists(waitingTime)
-        )
+private fun assertExistingOpenTabs(vararg tabTitles: String) {
+    for (title in tabTitles) {
+        try {
+            assertTrue(
+                mDevice.findObject(
+                    UiSelector()
+                        .resourceId("$packageName:id/mozac_browser_tabstray_title")
+                        .textContains(title)
+                ).waitForExists(pageLoadingTime)
+            )
+        } catch (e: AssertionError) {
+            println("The tab wasn't found")
+            mDevice.findObject(UiSelector().resourceId("$packageName:id/tabsTray")).swipeUp(2)
```

Comment:
Can you add a comment here on why `2` for future reading?

Reply:
I didn't write this method, I just modified it. But swipe methods take a maximum number of swipes, so I suppose who wrote this expected that 2 swipes up was enough to reach the end of tabs tray.

Reply:
removed that step